### PR TITLE
Removed home page.

### PR DESCRIPTION
**Pull request by YonatanNitzan**

### CODE REVIEW

File: src/routes/routes.js [hunk 1]
```diff
@@ -13,7 +12,7 @@ export default () => (
   <BrowserRouter>
     <Switch>
       <Route path="/scouting-form" exact component={ScoutingForm}/>
-      <Route path="/" exact component={HomePage}/>
+      <Route path="/" exact component={ScoutingForm}/>
```
Comment (TheArchNemesis) [inline src/routes/routes.js]: change the component to redirect on '/scouting-form'

Comment (YonatanNitzan) [reply src/routes/routes.js]: you mean component = <redirect path="/scouting-form">?

Comment (TheArchNemesis) [reply src/routes/routes.js]: <Redirect path="/scouting-form"/>

**Comment by YonatanNitzan** [comment]
Moved to #54

**Pull request closed without merging**